### ISSUE TITLE
polluted output (SyntaxWarning: invalid escape sequence '\*')

### ISSUE THREAD

**Issue by nchiapol** [spanezz/egt#13]
After updating to pyhon3.12 in trixie, egt started to corrupt my project-files with warnings

```
/usr/lib/python3/dist-packages/kitchen/text/converters.py:800: SyntaxWarning: invalid escape sequence '\*'
  '''Return a byte :class:`bytes` encoded so it is valid inside of any xml
/usr/lib/python3/dist-packages/kitchen/text/converters.py:832: SyntaxWarning: invalid escape sequence '\*'
  '''Decode a string encoded using :func:`bytes_to_xml`
```

I just investigated and the culprit is kitchen (see https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1076949).

Kitchen is used by taskw in `/usr/lib/python3/dist-packages/taskw/warrior.py` in case of errors.

For my own use I have just dropped the dependency on kitchen in that file. [Patch attached](https://github.com/user-attachments/files/16363452/drop_kitchen.patch.txt)

(This issue is intended as note in case others hit the same problem.)

**Comment by spanezz** [comment]
It could be that the fix or approach in #14 could help you here

**Comment by nchiapol** [comment]
the taskw package has dropped the dependency on kitchen as well (with `2.0.0-3` from 29 Aug 2024)

So no local patch needed anymore.
I am closing this issue.
